pkg/user/handler.go

```diff
@@ -247,7 +248,7 @@ func (h Handler) SignIn(c *gin.Context) {
 	}
 
 	var request signInRequest
-	if err := handler.DataBinder(c, &request); err != nil {
+	if err := handler.DataBinder(c, &request); err != nil && err != io.EOF {
```

Comment:
why is that needed?

Reply:
Because that's the error which is returned if the request body is nil